pkg/actor/system.go

```diff
@@ -566,6 +574,9 @@ func (s *System) poll(ctx context.Context, id int) {
 			if !running {
 				// Close the actor and mailbox.
 				p.onActorClosed()
+				log.Info("actor is closed",
+					zap.Uint64("id", uint64(p.mb.ID())),
+					zap.String("name", s.name))
```

Comment:
We need to add the log after `p.onMailboxEmpty` too.

Reply:
It may be very verbose, when there are many tables.

Reply:
removed this log